administration/src/bp-modules/EmailLink.tsx

```diff
@@ -0,0 +1,6 @@
+import React, { ReactElement } from 'react'
+
+import { isEmailValid } from './applications/utils/verificationHelper'
+
+export const EmailLink = (email: string): ReactElement =>
```

Comment:
please convert this in a proper react functional component with a default export as we do for other functional components

Reply:
Nice catch ...